src/service-api.ts

```diff
@@ -111,6 +114,48 @@ const plugin: FastifyPluginAsync<GraaspImportZipPluginOptions> = async (fastify,
     return newItems;
   };
 
+  const addFileToZip = async (item: Item, dirPath, archive, member) => {
```

Comment:
I would consider passing more args (handler, createGetChildrenTaskSequence function, etc) to actually get this function out in an util file. It would make the service-api clearer and also allows us to unit test this function standalone.